### PR TITLE
Integration of chrome profile

### PR DESCRIPTION
**Pull request by Oytizzo**
## PR Type

- [x]Bug Fix & Feature

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made.
- [ ] Version number has been updated.
- [ ] Required modules have been added to respective "requirements*.txt" files.
- [x] Relevant Test Cases added to this description (below).
- [ ] (Team) Label with affected action categories and semver status.

## Overview

With this improvement we will be able to open chrome with pre saved profile
Also able to add extensions

In order to open existing chrome profile all the other chrome instance must be closed entirely.

## Test Cases
- [TEST-3259](https://qa.automationsolutionz.com/Home/ManageTestCases/Edit/TEST-3259/)
- [TEST-5696](https://zeuz.zeuz.ai/Home/ManageTestCases/Edit/TEST-5696/)

### CODE REVIEW

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py [hunk 1]
```diff
@@ -789,6 +804,13 @@ def Go_To_Link(step_data, page_title=False):
                     # any other shared capabilities can be added from the selenium document
                     capabilities[left.strip()] = right.strip()
 
+            # Todo: profile, argument, extension, chrome option => go_to_link
+            elif mid.strip().lower() in ("chrome option", "chrome options") and dependency["Browser"].lower() == "chrome":
+                browser_options.append([left, right])
```
Comment (Muntasib-creator) [inline Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: append left.lower() and replace (" ", "_", "-") with ""

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py [hunk 2]
```diff
@@ -789,6 +804,13 @@ def Go_To_Link(step_data, page_title=False):
...
+
+        print("Got these browser_options")
```
Comment (Muntasib-creator) [inline Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: remove prints or convert them into execlog() func

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py [hunk 3]
```diff
@@ -3810,48 +3832,23 @@ def upload_file_through_window(step_data):
         return CommonUtil.Exception_Handler(sys.exc_info(), None, "Error parsing dataset")
 
     try:
-        if platform.system() == "Darwin":
```
Comment (Muntasib-creator) [inline Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: Why this function is reverted? please fix this

**Review by Muntasib-creator — CHANGES_REQUESTED** [review]

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py (hunk 3 above)
Comment (Oytizzo) [reply Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: fixed

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py (hunk 2 above)
Comment (Oytizzo) [reply Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: used execlog()

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py (hunk 1 above)
Comment (Oytizzo) [reply Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: this has been done on line 788. doing this again would be repetitive. did right.strip()

**Review by Muntasib-creator — APPROVED** [review]

**Pull request merged**